Scripts/developer_scripts/cgal_create_release_with_cmake.cmake

```diff
@@ -200,7 +217,7 @@ if (TESTSUITE)
     endif()
   endforeach()
   file(REMOVE_RECURSE "${release_dir}/tmp")
-endif()
+endif() //TESTSUITE
```

Comment:
Comments in CMake start with `#`, and not `//`. Does not it give you a syntax error?!

Reply:
woops